src/routes/diluents.test.js

```diff
@@ -0,0 +1,49 @@
+import express from 'express';
+import request from 'supertest';
+import passport from 'passport';
+import AnonymousStrategy from 'passport-anonymous';
+
+import diluents from './diluents';
+import database from '../modules/database';
+
+/* eslint-disable camelcase */
+describe('diluents route resource', () => {
+  const app = express();
+
+  passport.use(new AnonymousStrategy());
+  app.use(diluents);
```

Comment:
If the `body-parser` middleware isn't required here, maybe it isn't required in the other tests?

Reply:
Seems to only be required if there is a POST or PUT request.

Reply:
Ah ok makes sense. So the "list of items" have only GET and therefore don't need it. Cool.